modules/@apostrophecms/doc-type/index.js

```diff
@@ -502,6 +502,23 @@ module.exports = {
         }
         return self.apos.migration.addSortify(self.__meta.name, { type: self.name }, field);
       },
+      // Add the "cacheInvalidatedAt" field to the documents that do not have it yet,
+      // and set it to equal doc.updatedAt.
+      // The migration will apply only for the documents of the same type as
+      // the modules' that use this method, via `self.apos.migration.add`.
+      addCacheMigration() {
+        self.apos.migration.eachDoc({ type: self.__meta.name }, 5, async doc => {
```

Comment:
should we exclude the `@apostrophecms/*` pieces?

Reply:
should we exclude drafts?

Reply:
No, there's no harm in being consistent and someone else might decide to make a piece-page for them.